typescript/sdk/src/utils/wagmi.ts

```diff
@@ -1,26 +1,26 @@
 import type { Chain as WagmiChain } from '@wagmi/chains';
 
-import type { ChainMetadata } from '../consts/chainMetadata';
-import { Testnets } from '../consts/chains';
+import { ChainMetadata, etherToken } from '../consts/chainMetadata';
+import { CoreChainName, Testnets } from '../consts/chains';
 
 export function chainMetadataToWagmiChain(metadata: ChainMetadata): WagmiChain {
   return {
-    id: metadata.id,
-    name: metadata.displayName,
+    id: metadata.chainId,
+    name: metadata.displayName || metadata.name,
     network: metadata.name as string,
-    nativeCurrency: metadata.nativeToken,
+    nativeCurrency: metadata.nativeToken || etherToken,
```

Comment:
is this a wagmi default?

Reply:
Nope but Nam asked me to make nativeToken optional. On the off chance this causes an issue for a PI chain, the dev can just specify the token in their metadata config